private/buf/bufwire/bufwire.go

```diff
@@ -84,6 +87,131 @@ type ModuleConfig interface {
 	Module() bufmodule.Module
 	Config() *bufconfig.Config
 	Workspace() bufmodule.Workspace
+	WorkspaceConfig() *bufwork.Config
+}
+
+// BuildModuleFileSetOptionForTargetPaths returns the bufmodulebuild.BuildModuleFileSetOption
+// required for the given ModuleConfigs. This is required to support the --path filter.
+func BuildModuleFileSetOptionForTargetPaths(
```

Comment:
This doesn't feel like it should be in `bufwire`, but I'm not sure where it is used yet. The exposing of this feels like it is breaking an abstraction.

We need way more explanation here